test/address.spec.ts

```diff
@@ -110,6 +110,7 @@ describe('address', () => {
             t.unconfidentialAddress,
             (f.base58check || f.bech32)!,
           );
+          assert.strictEqual(t.scriptPubKey!.toString('hex'), f.script);
```

Comment:
the script in the fixture is in ASM format, you have to parse it to hex to make the tests pass